src/native/external/patches/zlib-intel/0001-Make-zlib-intel-compile-clean-against-C4244.patch

```diff
@@ -0,0 +1,75 @@
+From edabaf799fd071a328e0adb743a98628df6649f0 Mon Sep 17 00:00:00 2001
```

Comment:
Why is a patch file needed in a PR?

Reply:
It makes it easier to separate our customizations so when we import a new upstream version we can just re-apply the patch.